### DISCUSSION TITLE
Missing Enable GPU harvesting

### DISCUSSION THREAD

**Discussion by JaroslavHorky** [Chia-Network/chia-blockchain#17942] [Support]
Hello,
I have a clean install of Ubuntu 22.04.4 LTS.
I have a fresh install of Chia Blockchain 2.2.1
I have an RTX 3060 GPU with the latest 550.78 drivers

When I go to Settings / Harvester in the Chia blockchain (GUI), the Enable GPU harvesting option is missing. What could it be?

I also tried changing this to true in the config.
   use_gpu_harvesting: false

I don't know how to deal with it yet.
Thank you for your help.

**Comment by BrandtH22** [comment]
Hey @JaroslavHorky , as a heads up we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia).

Generally when the GPU harvesting options are not available in the GUI that means the chia client is not able to find a compatible GPU. 
Can you verify with the `nvidia-smi` command as to whether the GPU is being properly identified by the computer and which index it indicates the gpu is connected?

The `nvidia-smi` command can be run directly in terminal.

**Comment by BrandtH22** [comment]
No response from OP, please reopen the ticket if this issue persists
